### PR TITLE
Socket events scaffolding

### PR DESCRIPTION
**Pull request by ashfurrow**
I've got to shift focus to the new state-fetching strategy Craig outlined [here](https://github.com/artsy/prediction/issues/119), but I wanted to get this basic stuff in so that the UI can call the salesperson to do stuff, even if it doesn't do anything right now :stuck_out_tongue:

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">1 Error</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:no_entry_sign:</td>
      <td>No CHANGELOG changes made</td>
    </tr>
    
  </tbody>
</table>
  

  
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%">2 Warnings</th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td>Needs testing on a Phone if change is non-trivial</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td><h4>Found unstubbbed networking requests</h4>

<ul>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
<li><a href="https://github.com/search?q=ARFairFavoritesNetworkModel+repo%3Aartsy%2Feigen&amp;ref=searchresults&amp;type=Code&amp;utf8=%E2%9C%93">ARFairFavoritesNetworkModelSpec</a> in <code>test_ignores_artworks_without_a_partner</code></li>
</ul>
</td>
    </tr>
    
  </tbody>
</table>
  

  


<p align="right" data-meta="generated_by_danger" data-base-commit="" data-head-commit="" >
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by orta** [comment]
:+1: Cool

File: Artsy/Networking/Live_Auctions/LiveAuctionSocketCommunicator.swift [hunk 1]
```diff
@@ -10,12 +10,14 @@ func on(event: SocketEvent, callback: [AnyObject] -> Void) -> NSUUID
 }
 
 @objc protocol LiveAuctionSocketCommunicatorDelegate: class {
-    // TODO: this needs to be broken up into smaller pieces. See: https://github.com/artsy/causality/issues/36
     func didUpdateAuctionState(state: AnyObject)
 }
 
 protocol LiveAuctionSocketCommunicatorType {
     weak var delegate: LiveAuctionSocketCommunicatorDelegate? { get set }
```
Comment (orta) [inline Artsy/Networking/Live_Auctions/LiveAuctionSocketCommunicator.swift]: one delegate won't be enough I think

**Pull request merged**